packages/core/src/components/views/CredentialDetailPrimaryHeader.tsx

```diff
@@ -30,6 +31,10 @@ const CredentialDetailPrimaryHeader: React.FC<CredentialDetailPrimaryHeaderProps
   const { ColorPalette } = useTheme()
   const { width, height } = useWindowDimensions()
   const isBranding11 = brandingOverlayType === BrandingOverlayType.Branding11
+
+  // Use cached schema name as fallback if no specific overlay name is found
```

Comment:
nit: I don't think this comment is needed here, probably suffices to have a similar comment in the actual getEffectiveCredentialName function itself

Reply:
removed